LearnWithMentorDAL/EF/LearnWithMentorInitializer.cs

```diff
@@ -60,6 +60,7 @@ private static void InitializeUsers(LearnWithMentor_DBEntities context)
             context.Database.ExecuteSqlCommand("DBCC CHECKIDENT('Users', RESEED, 0)");
             var users = new List<User>
             {
+            	//mentors
```

Comment:
something is wrong with the alignment, please align with the `new User` below, so that they start on the same column

Reply:
thanks for noticing this.
In IDE and my Sublime everything was good. So i have no idea what caused wrong alignment but i fixed it.